src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,347 @@
+import React, {
+  createElement,
+  ForwardRefExoticComponent,
+  ReactElement,
+  ReactNode,
+  RefObject,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import classnames from 'classnames'
+import { isElementInViewport } from './utils'
+
+type TooltipProps<T> = {
+  label: string
+  position?: 'top' | 'bottom' | 'left' | 'right' | undefined
+  className?: string
+  dataClasses?: string
```

Comment:
so I tried this out and actually React shows an error in the console because of a potential naming conflict with native `data-` attributes. I think this prop should be renamed to `wrapperClasses`.

![image](https://user-images.githubusercontent.com/2723066/111005165-ec986f00-834f-11eb-9cfa-b03633c0baef.png)